src/components/Tool/FixEmailAddresses/FixEmailAddresses.tsx

```diff
@@ -224,6 +263,45 @@ export const FixEmailAddresses: React.FC<FixEmailAddressesProps> = ({
     });
   };
 
+  const handleBulkConfirm = async () => {
+    const dataToSend = determineBulkDataToSend(dataState, defaultSource ?? '');
```

Comment:
```suggestion
    const dataToSend = determineBulkDataToSend(
      dataState,
      defaultSource === appName ? 'MPDX' : defaultSource ?? '',
    );
```

Reply:
Is the source still going to be `MPDX` when it is a non-Cru org? Seems like that is going to cause problems.

Reply:
I see what the mailing address logic is doing now and will update this code to do the same to catch any bad data.